app/models/state_file_base_intake.rb

```diff
@@ -78,16 +78,21 @@ def self.selected_intakes_for_deadline_reminder_notifications
       .where(efile_submissions: { id: nil })
       .where.not(df_data_imported_at: nil)
       .has_verified_contact_info
-      .select(&:should_not_be_sent_reminder)
```

Comment:
i renamed this because we are selecting the people who should be sent the reminder? i think??